cmd/http-tracer.go

```diff
@@ -130,10 +134,6 @@ func Trace(f http.HandlerFunc, logBody bool, w http.ResponseWriter, r *http.Requ
 		Headers:  reqHeaders,
 		Body:     reqBodyRecorder.Data(),
 	}
-	rw := logger.NewResponseWriter(w)
-	rw.LogBody = logBody
-	f(rw, r)
-
```

Comment:
We cannot do this for all operations @poornas we should selectively do this such that we don't overwhelm the trace output.

Reply:
this PR just fixes a bug where the request body is not being set in the trace request info because the http handler needs to process the request before the reqBodyRecorder can get hold of the buffer. Unless logBody is true, it will not trace the request body